children/README.md

```diff
@@ -14,16 +14,15 @@
 - **Dependencies**: [STAC API - Core](../core)
 
 A STAC API Landing Page (a Catalog) can return information about the Catalog and Collection child objects
-it contains using the link relation `children` to an endpoint `/children`. The purpose of this endpoint is
-to present a single resource from which clients can retrieve
-all the immediate child objects of a Catalog, which may be Catalog or Collection objects.
-The `child` link relations in a Catalog already allow for describing these
-relationships, but require a client to retrieve each resource URL to find any information about the children
-(e.g., title, description), which can cause significant performance issues in user-facing applications.
-
-It is recommended that the Catalog and Collection objects returned by the `/children` endpoint are the same
-as those referenced by `child` link relations in the STAC API Landing Page. Following these semantics, it should
-return the immediate children of the root Catalog, rather than all descendant catalogs or collections.
+it contains using the link relation `children` to an endpoint `/children`. The `/children` endpoint shall
```

Comment:
Just a general observation, but I think we mostly use MUST instead of SHALL in the documentation. Your recent PRs use SHALL and I think we should be more consistent here. Could we try to use MUST instead of SHALL? I just had to look it up again whether SHALL was equal to MUST or SHOULD and found it confusing.

Reply:
I just looked. We use must a lot more than shall. I'd prefer must also, since shall seems overly formal in English. I'll file an issue to make these consistent.